docs/installation-places.md

```diff
@@ -1,35 +0,0 @@
-# Installation
```

Comment:
ahhh, i'll add a section in places.md and installation.md about installing Places